content/english/java-basics/answer-key.md

```diff
@@ -63,13 +63,47 @@ x: 2.3 y: 2.3 z: 0
 ```
```

Comment:
Thanks for adding the answer key as well! Is this file hidden?

Reply:
Yes the file is hidden!